### PR TITLE
Fix double backspace in Safari

### PR DESCRIPTION
**Pull request by alexeyfrank**

### CODE REVIEW

File: src/xterm.js [hunk 1]
```diff
@@ -4782,4 +4786,4 @@
     Terminal.cancel = cancel;
 
     return Terminal;
-});
\ No newline at end of file
+});
```
Comment (parisk) [inline src/xterm.js]: Could you please enter a new line here to comply with the code styling of the repository?

**Comment by parisk** [comment]
Thanks for your PR @alexeyfrank :smile:.

I assume that this PR closes #28.

xterm.js seems to work fine in Safari when using backspace though. Can you help me reproduce this issue?

**Comment by alexeyfrank** [comment]
@parisk sorry, i found a bug in my code. If our css accidentally make textarea in terminal hidden - width: 0, height: 0  - then we can't focus on this textarea in Safari.

**Comment by parisk** [comment]
@alexeyfrank sounds like this is a bug of your application and not of xterm.js. Are you okay with closing this PR if this is the case?

**Pull request closed without merging**